evergreen.yml

```diff
@@ -1409,68 +1408,92 @@ functions:
           # Logger tests will delete existing logs, in order to avoid destroying important logs from our local environment
           # the test is feature gated and will only run on evergreen.
           # Logger tests also have to run in isolation because other tests running simultaneously will "pollute" the log file the logger tests analyse.
-          cargo test --test connection_tests integration::test_driver_log_level --features evergreen_tests -- --nocapture &&
-          cargo test --package mongo-odbc-core --lib util::test_connection::test::bad_host --features bad_host -- --exact --nocapture
+          cargo test --test connection_tests integration::test_driver_log_level --features evergreen_tests -- --nocapture 
 
           EXITCODE=$?
 
           # Stop the local ADF
           ./resources/run_adf.sh stop
           exit $EXITCODE
+
+  "run ubuntu mongosqltranslate integration tests":
+    - command: s3.get
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        remote_file: mongosqltranslate/snapshot/libmongosqltranslate-linux.so
+        local_file: mongosql-odbc-driver/libmongosqltranslate.so
+        bucket: translators-connectors-releases
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongosql-odbc-driver
+        script: |
+          ${prepare_shell}
+          sudo mkdir -p /opt/mongodb/atlas-sql-odbc-driver/
```

Comment:
Now that Nathan's PR is in, you should be able to change the logic to get the latest snapshot /release version.

Reply:
I copied what was done in the JDBC Driver.

Reply:
never mind. I didn't see that patrick already did that lol

Reply:
ah, yes, I see the S3 get is above 👍 
It might be that Oliver's changes will break your integration tests because the driver will not look into that directory anymore.